windows-nsis/INSTALL-win32.txt

```diff
@@ -0,0 +1,62 @@
+GENERAL QUICKSTART FOR WINDOWS
+
+OpenVPN requires a configuration file and key/certificate files. You should 
+obtain these and save them to OpenVPN's configuration directory, usually 
+C:\Program Files\OpenVPN\config. Starting with OpenVPN 2.4-alpha1 you can also 
+put user-specific configurations to %USERPROFILE%\OpenVPN\config when using 
+OpenVPN GUI.
```

Comment:
The end result of the [above email discussion](https://www.mail-archive.com/openvpn-devel@lists.sourceforge.net/msg12738.html) was that %USERPROFILE%\OpenVPN\config is probably the least bad solution for now. The "correct" place for the config files would be under %USERPROFILE%\Appdata, but that directory is hidden on the filesystem level, so users would not see it, even though it _is_ accessible, e.g. using _cd $HOME\Appdata_ in Powershell. The idea behind hiding the directory probably was that applications, not users, would directly modify that directory. So we'll go with the original plan.